python/delta_sharing/tests/test_delta_sharing.py

```diff
@@ -484,32 +485,229 @@ def test_get_table_protocol(profile_path: str):
             ),
             id="Google Cloud Storage",
         ),
+    ],
+)
+def test_load_as_pandas_success(
+    profile_path: str,
+    fragments: str,
+    limit: Optional[int],
+    version: Optional[int],
+    expected: pd.DataFrame
+):
+    pdf = load_as_pandas(f"{profile_path}#{fragments}", limit, version, None)
+    pd.testing.assert_frame_equal(pdf, expected)
+
+
+@pytest.mark.skipif(not ENABLE_INTEGRATION, reason=SKIP_MESSAGE)
+@pytest.mark.parametrize(
+    "fragments,limit,version,expected",
+    [
+        pytest.param(
+            "share8.default.deletion_vectors_with_dvs_dv_property_on",
+            None,
+            None,
+            pd.DataFrame(
+                {
+                    "id": [3, 4, 5, 6, 7, 8, 9],
+                    "value": ["3", "4", "5", "6", "7", "8", "9"],
+                    "timestamp": [
+                        pd.Timestamp("2023-05-31T18:58:33.633+00:00"),
+                        pd.Timestamp("2023-05-31T18:58:33.633+00:00"),
+                        pd.Timestamp("2023-05-31T18:58:33.633+00:00"),
+                        pd.Timestamp("2023-05-31T18:58:33.633+00:00"),
+                        pd.Timestamp("2023-05-31T18:58:33.633+00:00"),
+                        pd.Timestamp("2023-05-31T18:58:33.633+00:00"),
+                        pd.Timestamp("2023-05-31T18:58:33.633+00:00"),
+                    ],
+                    "rand": [
+                        0.7918174793484931,
+                        0.9281049271981882,
+                        0.27796520310701633,
+                        0.15263801464228832,
+                        0.1981143710215575,
+                        0.3069439236599195,
+                        0.5175919190815845,
+                    ],
+                }
+            ),
+            id="dv",
+        ),
+        pytest.param(
+            "share8.default.deletion_vectors_with_dvs_dv_property_on",
+            6,
+            None,
+            pd.DataFrame(
+                {
+                    "id": [3, 4, 5, 6, 7, 8],
```

Comment:
How did you confirm that the returned dataframe is correct?

Reply:
i printed it out on dogfood